### PR TITLE
fix issue with "unexpected end of file" in health checks on subdomain test

### PR DESCRIPTION
**Pull request by kwypchlo**
- increase proxy timeouts to a higher than expected value on nginx endpoints that proxy to a different nginx location and let the location that proxies to an external resource manage the actual timeouts
- remove `/` in front of `$request_uri` because it already contains it and results in double slash otherwise

### CODE REVIEW

**Review by peterjan — APPROVED** [review]
Are we including this `proxy-pass-internal` in routes where we don't overwrite the timeouts? If so this PR is drastically changing the behaviour for those routes and we should verify whether that behaviour is wanted in all cases where that is true.

**Comment by kwypchlo** [comment]
> timeouts

default proxy timeouts apply either way so we're not changing anything on those finishing locations
http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_read_timeout

**Comment by kwypchlo** [comment]
> Are we including this `proxy-pass-internal` in routes where we don't overwrite the timeouts? If so this PR is drastically changing the behaviour for those routes and we should verify whether that behaviour is wanted in all cases where that is true.

default proxy timeouts apply either way so we're not changing anything on those finishing locations
http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_read_timeout

File: docker/nginx/conf.d/client.conf [hunk 1]
```diff
@@ -480,19 +484,21 @@ server {
 		}
 
 		proxy_set_header Dnslink-Lookup "";
-		proxy_pass http://127.0.0.1/$dnslink/$request_uri;
+		proxy_pass http://127.0.0.1/$dnslink$request_uri;
```
Comment (ro-tex) [inline docker/nginx/conf.d/client.conf:487]: Do we ever verify that the `dnslink` ends with a slash? I see we're grabbing it from the headers in 
```
ngx.var.dnslink = headers["skynet-skylink"]
```
but I'm not sure we can rely on that being always valid.

Can we verify that it's either empty or ends with a slash?

**Review by ro-tex — APPROVED** [review]
LGTM, as long as the concern about the potential missing trailing slash is unfounded.

File: docker/nginx/conf.d/client.conf (hunk 1 above)
Comment (kwypchlo) [reply docker/nginx/conf.d/client.conf:487]: it's not that $dnslink ends with slash, it's $request_uri that always starts with one :)

**Pull request merged**